### DISCUSSION TITLE
NSGA-II and how it works

### DISCUSSION THREAD

**Discussion by RafaHPSUnicamp** [community/community#76027] [Programming Help]
### Body

Hi, I need to know better how to understand how NSGA-II and NSGA-III fill the values of variables. The reason of this, is because i have a code that fill the values of all 8 values, but looking in a more logical way, makes more sense that the first 4 variables have to filled, rather the last 4. It is because the first variables have 0 cost to use, and the last 4 variables have a cost (which deduces in the profit). The goal is to maximize profit and mimimize areas. The variables are related to areas, and both equations to maximization and minimization is related to areas.

So, it is normal to NSGA-II and NSGA-III try to fill all 8 variables simultaneously?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MegatIrfan** [comment]
Yes, it is normal for NSGA-II and NSGA-III to optimize all variables simultaneously in a multi-objective optimization problem. 👍
